jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/ContainerParameters.java

```diff
@@ -107,6 +108,23 @@ public void setEnvironment(Map<String, String> environment) {
     this.environment = environment;
   }
 
+  @Input
+  @Nullable
+  @Optional
+  public List<String> getExtraClasspath() {
+    if (System.getProperty(PropertyNames.CONTAINER_EXTRA_CLASSPATH) != null) {
+      // TODO: this should probably support Java-style ":" or ";" separators
```

Comment:
Should we support `;`? That may become confusing if we allow passing `/d1:/d2:/d2` as-is as a string.

Reply:
Maybe we should just remove this comment, since we will take the property as-is as a string as discussed above.